int64 pose_x
int64 pose_y
int64 size_x
int64 size_y
string id
